roles/openshift_node/tasks/dnsmasq_install.yml

```diff
@@ -12,23 +12,18 @@
 
 - when: not openshift_is_atomic | bool
   block:
+  - name: Ensure dbus is updated before installing dnsmasq
+    package:
+      name: dbus
+      state: installed
+    register: dbus_update
+    until: dbus_update is succeeded
```

Comment:
I think we should just do nothing with dbus and forego this task as well.

Reply:
Done